html/418-request-denied.shtml

```diff
@@ -0,0 +1,15 @@
+<!--#if expr="${VERBOSE_ERROR_PAGES}=true" -->
+<p>
+    Sorry, we are refusing to process your request. If you believe we have
+    made a mistake please raise a support ticket with the following
+    information:
+</p>
+<code style="display: block;">
+  ORIG URL: <!--#echo var="http_x_orig_url" --><br />
+  ORIG ARGS: <!--#echo var="http_x_orig_args" --><br />
+  CODE: <!--#echo var="http_x_naxsi_sig" --><br />
+  TIME: <!--#config timefmt="%Y-%m-%dT%H:%M:%S %Z" --><!--#echo var="date_local" -->
+</code>
+<!--#else -->
+  <p>We have been unable to process the information you tried to submit.</p>
```

Comment:
In order to maintain the status quo for this image, the else case should be empty. We can have something nicer in the GovUK proxy.